__init__.py

```diff
@@ -2,32 +2,32 @@
 
 
 import base64
-import unicodedata
 import json
+import unicodedata
 from pathlib import Path
 
+from anki import version as ankiversion
 from anki.hooks import addHook, wrap
 from anki.utils import htmlToTextLine
-from anki import version as ankiversion
-
 from aqt import mw
-from aqt.qt import QClipboard
 from aqt.editor import Editor
+from aqt.qt import QClipboard
 from aqt.reviewer import Reviewer
-from aqt.utils import tooltip, showText
+from aqt.utils import showText, tooltip
 
+from .config import getUserOption
 from .semieditor import semiEditorWebView
 
-
-config = mw.addonManager.getConfig(__name__)
-
 ankiver_minor = int(ankiversion.split('.')[2])
 ankiver_major = ankiversion[0:3]
 
-#Get js files.
+# Get js files.
+
```

Comment:
There shouldn't be any blank lines here, as the above comment describes the function below